src/platform/datapath_raw_xdp_win.c

```diff
@@ -1589,12 +1574,11 @@ CxPlatXdpRx(
         // mark it resolved. This allows stateless sends to be issued without performing
         // a route lookup.
         //
-        Packet->Route->State = RouteResolved;
+        Packet->RecvData.Route->State = RouteResolved;
 
-        if (Packet->Buffer) {
-            Packet->Allocated = TRUE;
-            Packet->Queue = Queue;
-            Buffers[PacketCount++] = (CXPLAT_RECV_DATA*)Packet;
+        if (Packet->RecvData.Buffer) {
+            Packet->RecvData.Allocated = TRUE;
+            Buffers[PacketCount++] = &Packet->RecvData;
         } else {
             CxPlatListPushEntry(&Queue->PartitionRxPool, (CXPLAT_SLIST_ENTRY*)Packet);
```

Comment:
>CxPlatListPushEntry(&Queue->PartitionRxPool, (CXPLAT_SLIST_ENTRY*)Packet); [](http://example.com/codeflow?start=12&length=74)

This is wrong now.

Reply:
Talked over IM. We are reinterpreting the packet memory as a list entry, so it should be fine so long as we don't use both at the same time.